app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2>Sign up</h2>
+
```

Comment:
- [ ] [OPTIONAL] 
I think, we can improve upon the signup form by ensuring that the _password confirmation_ input field has the same dimension as the other input fields.

**Let's improve upon the dimension of the password confirmation input field:**
![image](https://user-images.githubusercontent.com/12745474/231575393-66f2f4fc-b27d-4451-aa84-0cbd1a150e64.png)